README.md

```diff
@@ -6,60 +6,42 @@ docker/cli
 This repository is the home of the cli used in the Docker CE and
 Docker EE products.
 
-It's composed of 3 main folders
-
-* `/cli` - all the commands code.
-* `/cmd/docker` - the entrypoint of the cli, aka the main.
-
 Development
 ===========
 
-### Build locally
+`docker/cli` is developed using Docker.
 
-```
-$ make build
-```
+Build a linux binary:
 
 ```
-$ make clean
+$ make -f docker.Makefile binary
```

Comment:
I would be fine with having a `build.sh` script instead. Not for this PR, just my 2¢.